src/zcl_abapgit_repo.clas.abap

```diff
@@ -139,6 +139,8 @@ CLASS zcl_abapgit_repo DEFINITION
       RAISING
         zcx_abapgit_exception .
     METHODS reset_status .
+    METHODS validate_branch
```

Comment:
can we change this to not be online specific? if its named `validate` then it can also do other things in the future, plus potentially something for offline repos

sidenote: need to restart work on getting repo under a interface sometime

Reply:
makes sense